hw02/hw02.py

```diff
@@ -100,4 +113,11 @@ def make_repeater(f, n):
     390625
     """
     "*** YOUR CODE HERE ***"
-
+    x=5
+    k=0
+    while k < n:
+         x = f(x)
+         k+=1
+    return x
+
+print(make_repeater(3,square))
```

Comment:
You don't need to call print in solution. Please read the requirement:

"Returns the function that computes the nth application of f."

Is your `make_repeater` returns function?